docs/supported-languages.md

```diff
@@ -76,10 +137,123 @@ The following thresholds define each maturity level:
 
 Visit the cheat sheet generation script and associated semgrep-core test files to learn more about each feature:
 
-* [generation script](https://github.com/returntocorp/semgrep/blob/develop/scripts/generate_cheatsheet.py)
-* [semgrep-core test files](https://github.com/returntocorp/semgrep/tree/develop/semgrep-core/tests)
-## Language parse rates
+* [Generation script](https://github.com/returntocorp/semgrep/blob/develop/scripts/generate_cheatsheet.py)
+* [`semgrep-core` test files](https://github.com/returntocorp/semgrep/tree/develop/semgrep-core/tests)
+
+### Language parse rates
+
+See [Parse rates by language](https://dashboard.semgrep.dev/).
+
+## Semgrep Supply Chain
+
+<SscIntro/>
+
+Semgrep Supply Chain parses **lockfiles** for dependencies, then scans your codebase for reachable findings based on the lockfiles. Some languages, such as Java, have several lockfiles, depending on your repository's package manager. For some languages, such as Javascript and Python, a manifest file is also parsed.
+
+The following tables list lockfiles, their package manager and language, and their support levels.
+
+### General Availability
+
+<table>
+<thead><tr>
+    <th>Language</th>
+    <th>Supported package managers</th>
+    <th>Lockfile</th>
+</tr></thead>
+<tbody><tr>
+   <td>Go</td>
+   <td>Go modules (<code>go mod</code>)</td>
+   <td><code>go.sum</code></td>
+  </tr>
+  <tr>
+   <td rowspan="2">JavaScript / TypeScript</td>
+   <td>npm (Node.js)</td>
+   <td><code>package-lock.json</code></td>
+  </tr>
+  <tr>
+   <td>Yarn, Yarn 2, Yarn 3</td>
+   <td><code>yarn.lock</code></td>
+  </tr>
+  <tr>
+   <td rowspan="2">Python</td>
+   <td>pip</td>
+   <td><code>Pipfile.lock</code></td>
+  </tr>
+  <tr>
+   <td>Poetry</td>
+   <td><code>poetry.lock</code></td>
+  </tr>
+  <tr>
+   <td>Ruby</td>
+   <td>RubyGems</td>
+   <td><code>Gemfile.lock</code></td>
+  </tr></tbody>
+</table>
+
+### Beta
+
+<table>
+  <thead><tr>
+   <th>Language</th>
+   <th>Supported package managers</th>
+   <th>Lockfile</th>
+  </tr></thead>
+  <tbody><tr rowspan="2">
+   <td rowspan="2">Java</td>
+   <td>Gradle</td>
+   <td><code>gradle.lockfile</code></td>
+  </tr>
+  <tr>
+   <td>Maven (single-file configurations)</td>
+   <td><code>pom.xml</code>*</td>
+  </tr></tbody>
+</table>
+
+
+_* `pom.xml` is not a lockfile, however Semgrep Supply Chain parses `pom.xml` as the source of truth for dependencies in Maven projects in conjunction with `MANIFEST.mf`._
```

Comment:
May we also add another caveat below to include 'requirements.txt' as well? It's like the precursor to the lockfile

Reply:
Indeed we can. Added!